src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsEventSourceLifetime.cs

```diff
@@ -21,6 +21,7 @@ public class TestsEventSourceLifetime
         /// </summary>
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // non-Windows EventSources don't have lifetime
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "ExerciseEventSource use private member reflection")]
```

Comment:
@tijoytom, thanks for the pointer.  Is there a recommended way of implementing this test on UAP?  The test itself is still valid on UAP.